rpc/nodeagent/compute/virtualmachine/virtualmachine.proto

```diff
@@ -52,13 +52,31 @@ message UserConfiguration {
 	UserType usertype = 3;
 }
 
+message RDPConfiguration {
+	bool disableRDP = 1;
+}
+
+message LinuxConfiguration {
+	bool disablePasswordAuthentication = 1;
+}
+
+message WindowsConfiguration {
+	bool enableAutomaticUpdates = 1;
+	string timeZone = 2;
+	RDPConfiguration RDPConfiguration = 3;
+}
+
 message OperatingSystemConfiguration {
 	string computerName = 1;
 	UserConfiguration administrator = 2;
 	repeated UserConfiguration users = 3;
 	string customData = 4;
 	repeated SSHPublicKey publickeys = 5;
 	OperatingSystemType ostype = 6;
+	// bootstrap engine can be cloud-init, Windows answer files, ...
```

Comment:
my money is that our next support will be ignition. https://github.com/coreos/ignition